cmake/stm32/f3.cmake

```diff
@@ -4,19 +4,19 @@ set(STM32_F3_TYPES
     F378xx F398xx 
 )
 set(STM32_F3_TYPE_MATCH 
-    "301.[68]" "302.[68]" "302.[BC]" "302.[ED]" "303.[68]" "303.[BC]" 
-    "303.[ED]" "318.." "328.." "334.[468]" "358.." "373.[8BC]"
+    "301.[68]" "302.[68]" "302.[BC]" "302.[DE]" "303.[68]" "303.[BC]" 
+    "303.[DE]" "318.." "328.." "334.[468]" "358.." "373.[8BC]"
     "378.." "398.."    
 )
 set(STM32_F3_RAM_SIZES 
-     16K  16K   0K  64K  16K   0K  
-     80K  16K  16K  16K  48K   0K
-     32K  80K
+     16K  16K   0K  64K  12K   0K  
```

Comment:
From my understanding the third (302.[BC]) is 40Ko
![image](https://user-images.githubusercontent.com/29129731/102714691-b7d8c680-42d0-11eb-8c23-260b2d9eb91e.png)

Reply:
see #169 and look at bottom of file. I know there is zero but in fact it's implemented. You will see easier with raw files instead diff ;)